ci/run_truffle_tests.sh

```diff
@@ -54,4 +54,4 @@ $(npm bin)/truffle test $(find ../disputer/test -name '*.js') --network ci
 $(npm bin)/truffle test $(find ../monitors/test -name '*.js') --network ci
 
 # Only run tests that needs mainnet forking here
-$(npm bin)/truffle test ../liquidator/test-fork/ --network mainnet-fork
\ No newline at end of file
+$(npm bin)/truffle test $(find ../liquidator/test-fork -name '*.js') --network mainnet-fork
```

Comment:
Just curious -- what was the issue with the previous syntax?

Reply:
Just making it consistent with the other tests

Reply:
Gotcha. +1